docs/guides/testing/testing.rst

```diff
@@ -67,19 +66,6 @@ Integration Tests
 
 .. _Django test client: https://docs.djangoproject.com/en/dev/topics/testing/overview/
 
-
-UI Acceptance Tests
```

Comment:
I was wondering if this section should remain with some note about cypress tests, but I guess this reflects the current state for the Open edX platform. For edX.org, we still use https://github.com/edx/edx-e2e-tests, but I guess that repo remained with edX.org, so I am not sure how and if to document here.  We can reach out to tCRIL as well.

Reply:
In terms of Open edX in general: Yeah, unfortunately, we have no official recommendation to give for acceptance testing currently 😞 I haven't investigated Cypress much, but it sounds promising. If edX is having a good experience with it, it could be good to mention it somewhere as a starting point for other community members.

In terms of this PR: Cypress isn't being used in edx-platform, right? If not, then this probably isn't the place to mention it.

Reply:
Also, +1 to not mentioning edx-e2e-tests here (unless there is a plan to make them part of the openedx org).

Reply:
I agree, I think you don't need to mention Cypress or e2e at the moment.  Once we have more info we can always add things back.

Reply:
Thanks @kdmccormick and @feanil. 
@awais786: feel free to leave this as-is and it can be adjusted once there is anything more definitive to say about acceptance testing.